### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by Seviel**
Remove SFML stuff from examples and update them to use rinvid app and screens.

### CODE REVIEW

File: examples/sprites/main.cpp [hunk 1]
```diff
@@ -96,68 +74,35 @@ int main()
...
-        auto end   = std::chrono::high_resolution_clock::now();
-        delta_time = end - start;
+    if (state_changed == true)
+    {
+        current_animation_state = action + "_" + direction;
+        robot_sprite.play(current_animation_state);
     }
 
-    return 0;
+    robot_sprite.move(rinvid::Vector2<float>{horizontal_delta, 0});
```
Comment (zolalsl) [inline examples/sprites/main.cpp:98]: Maybe we should use pixels per second unit for movement? 

Same goes for other example

Comment (Seviel) [reply examples/sprites/main.cpp:98]: I would be ok with using pixels per second, but I don't think we should do it as part of this change. This change is about removing references to SFML from examples and using Rinvid's Aplication and Screen classes. That being said, if u want you can make another PR to make the change yourself if you wish.

Comment (zolalsl) [reply examples/sprites/main.cpp:98]: Yeah, that sounds like better idea.

**Review by zolalsl — APPROVED** [review]
LGTM

**Pull request merged**